---
C22_CompactGroundRecognitionAndMapping/C22C24_PATH drivingPath
